django/db/models/base.py

```diff
@@ -834,6 +834,13 @@ def save_base(self, raw=False, force_insert=False,
         # Once saved, this is no longer a to-be-added instance.
         self._state.adding = False
 
+        # Readonly fields are computed by the Database. If there are any,
+        # we should fetch their value.
+        if self._meta.has_readonly_fields:
```

Comment:
A whole new argument to save look a little overkill to me. The query is a simple one that should not cost much...

Plus, at some point in the future, we might use `RETURNING` so there won't be any additional query for supported backends...